### PR TITLE
[SRM-512] SDPAP-6730 Remove deprecated "drupal_get_path()"

### PR DESCRIPTION
**Pull request by mayurngondhkar**
**Jira:**
https://digital-vic.atlassian.net/browse/SRM-512
https://digital-vic.atlassian.net/browse/SDPAP-6730

**Changes**

1. Remove deprecated `drupal_get_path()` function

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
-        "dpc-sdp/tide_media": "^3.0.0",
-        "dpc-sdp/tide_webform": "^3.0.0"
+        "dpc-sdp/tide_core": "dev-feature/SRM-512_update_drupal_to_9.4",
+        "dpc-sdp/tide_media": "dev-feature/SRM-512_update_drupal_to_9.4",
+        "dpc-sdp/tide_webform": "dev-feature/SRM-512_update_drupal_to_9.4"
     },
     "suggest": {
-        "dpc-sdp/tide_api:^3.0.0": "Allows to use Drupal in headless mode"
+        "dpc-sdp/tide_api:dev-feature/SRM-512_update_drupal_to_9.4": "Allows to use Drupal in headless mode"
     },
```
Comment (MdNadimHossain) [inline composer.json]: revert these before merging

File: dev-tools.sh [hunk 2]
```diff
@@ -33,6 +33,6 @@
 # stable version.
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
-# export GH_COMMIT=COMMIT_SHA
+export GH_COMMIT=960eba39ed9543b17654dbd5e06f3a2ab2b1fb4b
```
Comment (MdNadimHossain) [inline dev-tools.sh]: revert it before merging

**Review by MdNadimHossain — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (mayurngondhkar) [reply composer.json]: Changes reverted

File: dev-tools.sh (hunk 2 above)
Comment (mayurngondhkar) [reply dev-tools.sh]: Changes reverted

**Pull request merged**